client/src/Movies/MovieCard.js

```diff
@@ -1,7 +1,32 @@
 import React from 'react';
+import { Link } from "react-router-dom";
 
 const MovieCard = props => {
-  return;
+const { title, director,metascore, stars } = props;
+return(
+  <div className="movie-card">
+  <Link to={`/movie/${props.id}`}>
+    <h2>{title}</h2>
+
+  <div className="movie-director">
+    Director: <em>{director}</em>
+  </div>
+
+  <div className="movie-metascore">
+    Metasccore: <strong>{metascore}</strong>
+  </div>
+
+  <h3>Actors</h3>
+
+  {stars.map(star => (
+    <div key={star} className="movie-star">
+    {star}
+    </div>
+    ))}
+    </Link>
+  </div>
+  
```

Comment:
Great job breaking out this function into its own component.
You rocked this project! 
🤘🏼 🌟🎸🥽